addons/website_forum/tests/test_forum.py

```diff
@@ -509,3 +510,13 @@ def test_forum_post_compose_message_without_branding(self):
         })
         post = self.env["forum.post"].search([('forum_id', '=', self.forum.id), ('name', '=', 'test_branding')])
         self.assertNotIn("data-oe-", post.message_ids.body)
+
+    def test_forum_post_link(self):
+        self.user_portal.karma = KARMA['tag_create']
+        post = self.env['forum.post'].with_user(self.user_portal).create({
+                'name': "Post Forum test",
+                'content': """<a href="https://www.example.com/" rel="ugc">test</a>""",
```

Comment:
Would we add a bit of "noise" around a link to make sure it works properly?

Suggestion: "This is a test link: [thelink]. Let's make sure it works.".